modules/node_modules/@ncigdc/components/GitHut.js

```diff
@@ -64,6 +69,22 @@ class GitHut extends React.Component {
     }
   }
 
+  handleAngularRouterRequestGo = (state, params, options) => {
```

Comment:
I am open to suggestions for what to name this
It handles when angular's router requests a ui-router state change